src/components/PopupWithForm/PopupWithForm.js

```diff
@@ -115,6 +169,7 @@ export default function PopupWithForm(props) {
               )}
               <button
                 type='submit'
+                onClick={isSignIn ? handleSignIn : handleSignUp}
```

Comment:
`                onClick={isSignIn ? handleSignIn : handleSignUp}
`

please, delete it fron the button


Submit handlers are added only to tags `form` with event `submit` rather than to submit buttons with event `click` because it also automatically handles `Enter` presses